src/backend/src/controllers/projects.controllers.ts

```diff
@@ -140,6 +140,48 @@ export default class ProjectsController {
     }
   }
 
+  static async createMaterial(req: Request, res: Response, next: NextFunction) {
+    try {
+      const {
+        name,
+        assemblyId,
+        status,
+        materialTypeName,
+        manufacturerName,
+        manufacturerPartNumber,
+        pdmFileName,
+        quantity,
+        unitName,
+        price,
+        subtotal,
+        linkUrl,
+        notes
+      } = req.body;
+      const creator = await getCurrentUser(res);
+      const wbsNum = validateWBS(req.params.wbsNum);
+      const id = await ProjectsService.createMaterial(
+        creator,
+        name,
+        status,
+        materialTypeName,
+        manufacturerName,
+        manufacturerPartNumber,
+        quantity,
+        unitName,
+        price,
+        subtotal,
+        linkUrl,
+        notes,
+        wbsNum,
+        assemblyId,
+        pdmFileName
+      );
+      return res.status(200).json({ message: `Successfully created material with id #${id}` });
```

Comment:
we've started just returning the created object instead of a message like this, so you should do that here